packages/shared/components/inputs/DateTimePicker.svelte

```diff
@@ -1,41 +1,43 @@
 <!-- this is a wrapper component for svelty-picker -->
 <script lang="ts">
     import SveltyPicker from 'svelty-picker'
-    import { createEventDispatcher } from 'svelte'
+    import sveltyPickerTranslations from 'svelty-picker/i18n'
+    import { ComponentProps, createEventDispatcher } from 'svelte'
     import { Tooltip, Button, ButtonSize } from 'shared/components'
     import { localize } from '@core/i18n'
-    import sveltyPickerTranslations from 'svelty-picker/i18n'
     import { appSettings } from '@core/app'
 
-    export let value: Date = undefined
-    export let startDate: Date = null
+    export let value: Date | undefined = undefined
+    export let startDate: Date | null = null
     export let mode: 'auto' | 'datetime' | 'date' | 'time' = 'auto'
 
     const dispatch = createEventDispatcher()
     const sveltyPickerStartDate = convertDateToSveltyPickerFormat(startDate)
-
-    let sveltyPickerDate = convertDateToSveltyPickerFormat(value) ?? sveltyPickerStartDate
     const translations = getSveltyPickerTranslations()
+
+    let sveltyPickerDate = value ? convertDateToSveltyPickerFormat(value) : sveltyPickerStartDate
     let tooltip: Tooltip
 
-    function convertDateToSveltyPickerFormat(date: Date): string {
-        return date?.toLocaleString('sv')
+    function convertDateToSveltyPickerFormat(date: Date | null): string | undefined {
+        return date ? date?.toLocaleString('sv') : undefined
```

Comment:
same here, no need to `date ? date?.`, eg
```suggestion
    function convertDateToSveltyPickerFormat(date: Date | undefined): string | undefined {
        return date?.toLocaleString('sv') ?? undefined
```